src/Plugin/Field/FieldWidget/MinisiteWidget.php

```diff
@@ -75,42 +55,26 @@ protected function formMultipleElements(FieldItemListInterface $items, array &$f
   public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
     $element = parent::formElement($items, $delta, $element, $form, $form_state);
 
-    $field_settings = $this->getFieldSettings();
```

Comment:
the logic here was a bit strange. 

What we want is to check that, if `file_validate_extensions` is one of our existing validators, then it should be using our own extensions to validate rather than defaults.